win32ss/gdi/ntgdi/region.c

```diff
@@ -3662,10 +3662,6 @@ NtGdiCreateRoundRectRgn(
         bottom = tmp;
     }
 
-    /* the region is for the rectangle interior, but only at right and bottom for some reason */
```

Comment:
We can observe that this piece of code has been added (and is still present) in Wine:
https://github.com/wine-mirror/wine/blame/b80ea4153b096970514f86f385a37c9ceaceada2/dlls/win32u/region.c#L704
from this commit https://github.com/wine-mirror/wine/commit/3e51dd751568730d500b7ed7402fbf308a99f028

Reply:
It's wrong. You can't have a balloon tooltip without it.

Reply:
Reverted change.